OrchardCore.Commerce/Views/TextProductAttributeField_Edit.cshtml

```diff
@@ -13,7 +11,7 @@
         @if (settings.Required) {
             <li class="hint">@T["Required"]</li>
         }
-        @if ((settings.PredefinedValues != null) && (settings.PredefinedValues.Count() > 0)) {
+        @if (settings.PredefinedValues != null && settings.PredefinedValues.Any()) {
```

Comment:
After seeing this pattern for the Nth time (or was it the Mth time?), I'm longing for an extension method `HasItems()` or `IsNotEmpty()` that will operate on `null`, too.

But we can tackle that in the next issue.

Reply:
That should definitely be in HL, and I don't mean HastLayer 😁